ossl/src/lib.rs

```diff
@@ -213,6 +213,15 @@ impl OsslContext {
         }
     }
 
+    /// Returns the default global context.
+    ///
+    /// See `OSSL_LIB_CTX_get0_global_default` for details.
+    pub fn global_default() -> OsslContext {
```

Comment:
This was intentionally not exposed, why do you need to make it public ?

Reply:
I am planning to augment that code to also allow you to load the default openssl configuration in it (it currently doesn't because kryoptic case is special and it explicitly shouldn't use the default config either).

Reply:
That works fine, thanks for the pointer!